test/support/samples/swagger_formater.ex

```diff
@@ -0,0 +1,176 @@
+defmodule Xcribe.Support.Samples.SwaggerFormater do
+  def path_item_object_without_request_body do
+    %{
+      "put" => %{
+        summary: "",
+        description: "",
```

Comment:
Shouldn't we try to test at least one of these fields with a configured value?

Reply:
At this point I'm not using the information módule to get descriptions (we need do that soon) so I can't do this test now.

Reply:
np. Do we have these TODOs mapped somewhere (maybe an issue to keep track of this)?

Reply:
Now we have  =D  https://github.com/brainn-co/xcribe/issues/22